frontend/src/lib/components/details/seasons.svelte

```diff
@@ -1,124 +1,64 @@
 <script lang="ts">
-  import ReadMore from "./read_more.svelte"
-  import MediaQuery from "svelte-media-query"
-  import { IMG_W342, IMG_W500 } from "../../variables"
+  import { Swiper, SwiperSlide } from "swiper/svelte"
+  import { Navigation } from "swiper"
+  import { IMG_W500 } from "$lib/variables"
```

Comment:
Consider using the W342, it's 342x513 and gets cached by Redis - We can also add caching to the W500 if we need to use them

Reply:
Done